packages/shared/src/components/tabs/TabList.tsx

```diff
@@ -126,13 +131,14 @@ function TabList<T extends string = string>({
               className.item,
               'relative p-2 py-4 text-center font-bold typo-callout',
               isActive ? '' : 'text-text-tertiary',
+              isAnchor && 'cursor-pointer',
             )}
             onClick={() => onClick(tab)}
             type="button"
```

Comment:
If anchor type button should not be there

Reply:
Removed